src/app/components/dynamic-list/__snapshots__/DynamicList.test.jsx.snap

```diff
@@ -4,6 +4,7 @@ exports[`DynamicList given props does it create a list matches the snapshot 1`]
 <div>
   <h1
     className="dynamic-list__title"
+    data-testid="dynamic-list-title"
```

Comment:
as above, you do not query this element by this id in any of your test at the moment. might by leftover or you forgot to include the file?